flips/20210824-fcl-tx-payer-service.md

```diff
@@ -0,0 +1,349 @@
+# FCL Transaction Payer Service
+
+| Status        | Proposed                                             |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [612](https://github.com/onflow/flow/pull/612)       |
+| **Author(s)** | Jeffrey Doyle (jeffrey.doyle@dapperlabs.com),        |
+| **Sponsor**   |                                                      |
+| **Updated**   | 2020-09-09                                           |
+
+## Objective
+
+There is a need for the developement of a suite of tools that will collectively
+act as a Transaction Fee Payer Service for use by applications built on Flow.
+
+Technically, this suite of tools will facilitate signing as the payer of transactions initiated
+by application's users with an account controlled by the application itself.
+
+From a functional perspective, the service would enable users of applications that employ
+these tools to be able to execute transactions, and thereby interact with Flow, without
+encurrign the burden of having to pay for their transaction fees themselves. This user experience
+is valuable for many reasons, including enabling users to engague with applications while not
+having _any_ FLOW in their account, or having to worry about having _enough_ FLOW to execute the 
+transactions they wish to execute.
+
+A Transaction Fee Payer Service would exist as a resusable codebase / set of
+packages & tools that could be used by the applications that desire it.
+
+## Motivation
+
+One of the many features of Flow is that transaction fees can be paid by an account
+separate from the authorizers of that transaction. With this feature, applications that wish to 
+provide "free" transactions for their users could opt to pay for their users' 
+transactions themselves. They would do this by producing a signature for the 
+transaction as the payer of that transaction. Upon receiving this transaction, 
+the Flow network would then deduct transaction fees from the payer, not any of 
+the authorizers.
+
+For a user to pay their transaction fees themselves, that user would have to have FLOW
+in their account. This means that should a first time Flow user attempt to engague with an application
+that does not employ a Transaction Payer Service, they would first need to acquire FLOW before
+engaguing with that application. This is an undesireable user experience, as it significantly raises
```

Comment:
'This means that should a first time Flow user attempt to engague with an application
that does not employ a Transaction Payer Service, they would first need to acquire FLOW before
engaguing with that application' 
change to: 
'This can lead to a situation where a first-time Flow user attempts to engage with an application that does not employ a Transaction Payer Service, and they would first need to acquire FLOW before engaging with that application.'